### PR TITLE
Web Apps: Make sure all appearance attributes support multiple attributes

### PR DESCRIPTION
**Pull request by orangejenny**
## Product Description
Followup for https://github.com/dimagi/commcare-hq/pull/33394

https://dimagi-dev.atlassian.net/browse/USH-3587

### Safety story
Pretty straightforward changes. Running through QA because it affects the web apps UI.

### Automated test coverage

A little bit. One test updated in this PR.

### QA Plan

https://dimagi-dev.atlassian.net/browse/QA-5555

### Rollback instructions

- [x] This PR can be reverted after deploy with no further considerations

### Labels & Review
- [x] Risk label is set correctly
- [x] The set of people pinged as reviewers is appropriate for the level of risk of the change

### CODE REVIEW

File: corehq/apps/cloudcare/static/cloudcare/js/form_entry/spec/entries_spec.js [hunk 1]
```diff
@@ -70,7 +70,7 @@ hqDefine("cloudcare/js/form_entry/spec/entries_spec", function () {
             var entry;
 
             questionJSON.datatype = constants.SELECT;
-            questionJSON.style = { raw: constants.MINIMAL };
+            questionJSON.style = { raw: constants.MINIMAL + " dummy" };
```
Comment (MartinRiese) [inline corehq/apps/cloudcare/static/cloudcare/js/form_entry/spec/entries_spec.js:73]: 👍

**Review by MartinRiese — APPROVED** [review]

**Review by Jtang-1 — APPROVED** [review]
Thanks for making these changes

File: corehq/apps/userreports/migrations/__pycache__/0010_add_invaliducrdata.cpython-39.pyc.5872653216
Comment (calellowitz) [inline corehq/apps/userreports/migrations/__pycache__/0010_add_invaliducrdata.cpython-39.pyc.5872653216:1]: This should not be in the PR, right?

Comment (Jtang-1) [reply corehq/apps/userreports/migrations/__pycache__/0010_add_invaliducrdata.cpython-39.pyc.5872653216:1]: Thanks for flagging - i'm not sure how that got in there

**Pull request merged**